cni/pkg/plugin/redirect.go

```diff
@@ -265,7 +264,7 @@ func NewRedirect(pi *PodInfo) (*Redirect, error) {
 		return nil, fmt.Errorf("annotation value error for value %s; annotationFound = %t: %v",
 			"kubevirtInterfaces", isFound, valErr)
 	}
-	if v, found := pi.ProxyEnvironments[options.DNSCaptureByAgent.Name]; found {
+	if v, found := pi.ProxyEnvironments["ISTIO_META_DNS_CAPTURE"]; found {
```

Comment:
magic string smells bad, add TODO for it?

Reply:
The non-magic string is what caused the extra 25mb in the first place :slightly_smiling_face:

Reply:
IC, so just add TODO, that would be a nice start for someone who begin to contribute to this project.